install/playbooks/roles/system-prepare/tasks/main.yml

```diff
@@ -203,8 +203,8 @@
 - name: Wait for the server to come back online
   when: aa_enabled.rc == 1
   tags: security
-  local_action:
-    module: wait_for_connection
+  delegate_to: localhost
```

Comment:
Much cleaner, IMHO

Reply:
Yes, true. I will update it in consequence.

Reply:
This one was already fixed, maybe the code is old?